integration-test/src/main/java/com/sequenceiq/it/cloudbreak/DiskTypeTests.java

```diff
@@ -0,0 +1,183 @@
+package com.sequenceiq.it.cloudbreak;
+
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.newway.CloudbreakTest;
+import com.sequenceiq.it.cloudbreak.newway.DiskTypes;
+import com.sequenceiq.it.cloudbreak.newway.TestParameter;
+import com.sequenceiq.it.cloudbreak.newway.cloud.AwsCloudProvider;
+import com.sequenceiq.it.cloudbreak.newway.cloud.AzureCloudProvider;
+import com.sequenceiq.it.cloudbreak.newway.cloud.GcpCloudProvider;
+import com.sequenceiq.it.cloudbreak.newway.cloud.OpenstackCloudProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class DiskTypeTests extends CloudbreakTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DiskTypeTests.class);
+
+    public DiskTypeTests() {
+    }
+
+    public DiskTypeTests(TestParameter tp) {
+        setTestParameter(tp);
+    }
+
```

Comment:
Tests which try platform as { null, "", "unknown" } would be possible improvement. not a high priority one, but in the future...